Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:21.04 AS build
+
+RUN dpkg --add-architecture i386 && \
+  apt-get update && \
+  apt-get install -y curl
+
+RUN curl -L -o /usr/local/bin/red https://static.red-lang.org/dl/auto/linux/red-latest && \
+  chmod +x /usr/local/bin/red
+
+# WORKDIR /opt/rebol
+
+# RUN curl -L -O http://www.rebol.com/downloads/v278/rebol-core-278-4-10.tar.gz
+# RUN tar -xzf rebol-core-278-4-10.tar.gz
+# RUN chmod +x rebol-core/rebol
+# RUN cp rebol-core/rebol /usr/local/bin/rebol
+
+# WORKDIR /opt/red
+
+# RUN curl -L -O https://github.com/red/red/archive/refs/tags/v0.6.4.tar.gz
+# RUN tar -xzf v0.6.4.tar.gz
+# RUN cd red-0.6.4
+# RUN echo 'Rebol[] do/args %red.r "-d -r --no-view %environment/console/CLI/console.red"' | /usr/local/bin/rebol +q -s
```

Comment:
I started with manually compiling, but then decided to try and get the official image to work.

Reply:
To get official image to work, you'll need to add `libgtk-3-0:i386` to `apt-get install`.

Reply:
How did you build the source? Just downloaded the master archive, extracted it, `cd`-ed into the extracted directory and ran:

```
echo 'Rebol[] do/args %red.r "-d -r --no-view %environment/console/CLI/console.red"' | /usr/local/bin/rebol +q -s
```
?

Reply:
> Just downloaded [...] and ran:
> [...]
> ?

Exactly.

Reply:
> There's one more reason why I pre-build a console. Red official builds are full toolings, and console gets compiled on first run, which can take some time (on my super-old core2duo it takes 5 minutes). But perhaps that's not a problem?

That's not really an issue, as we can do the compiling in the Dockerfile which is only done once. All individual test runs then can re-use the compiled code